src/KintoBase.js

```diff
@@ -1,7 +1,5 @@
-"use strict";
-
-import Collection from "./collection";
 import BaseAdapter from "./adapters/base";
+import Collection from "./collection";
```

Comment:
Oh, sorry. This happens automatically in my editor. In Atom and Visual Studio Code I have an auto import-sort plugin enabled.